### PR TITLE
Update MKLDNN to v1.1.1.

### PR DESCRIPTION
**Pull request by ayzhuang**

### CODE REVIEW

File: src/ngraph/runtime/cpu/mkldnn_utils.cpp [hunk 1]
```diff
@@ -1755,8 +1755,10 @@ bool runtime::cpu::mkldnn_utils::is_bf16_supported()
     }
     catch (const mkldnn::error& e)
     {
+        std::cout << "bf16 is not supported: " << e.message << std::endl;
```
Comment (jbobba) [inline src/ngraph/runtime/cpu/mkldnn_utils.cpp]: use NGRAPH_ERR

File: src/ngraph/runtime/cpu/mkldnn_utils.cpp [hunk 2]
```diff
@@ -1755,8 +1755,10 @@ bool runtime::cpu::mkldnn_utils::is_bf16_supported()
...
         return false;
     }
+    std::cout << "bf is supported\n";
```
Comment (jbobba) [inline src/ngraph/runtime/cpu/mkldnn_utils.cpp]: remove.

File: cmake/external_mkldnn_v1.cmake [hunk 3]
```diff
@@ -256,6 +256,7 @@ else()
         BINARY_DIR "${EXTERNAL_PROJECTS_ROOT}/mkldnn/build"
         INSTALL_DIR "${EXTERNAL_PROJECTS_ROOT}/mkldnn"
         EXCLUDE_FROM_ALL TRUE
+        BUILD_ALWAYS 1
```
Comment (jbobba) [inline cmake/external_mkldnn_v1.cmake]: why are we adding this?

File: src/ngraph/runtime/cpu/mkldnn_utils.cpp (hunk 1 above)
Comment (ayzhuang) [reply src/ngraph/runtime/cpu/mkldnn_utils.cpp]: I will revert this commit. This is for debugging mac.

**Pull request merged**